### PR TITLE
FestMerch activity made

### PR DESCRIPTION
**Pull request by Agrim1234**
Fest Merchandise Activity is made. It is as per recommended by tanmay. And also i have added browser class for customized chrome custom tabs.

### CODE REVIEW

File: app/src/main/java/com/macbitsgoa/events/aboutmac/AboutMacActivity.java [hunk 1]
```diff
@@ -16,10 +16,10 @@ protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_about_mac);
 
-        ViewPager aboutMacViewPager = (ViewPager) findViewById(R.id.about_mac_viewpager);
+        ViewPager aboutMacViewPager = findViewById(R.id.about_mac_viewpager);
```
Comment (Rushi98) [inline app/src/main/java/com/macbitsgoa/events/aboutmac/AboutMacActivity.java]: why these changes are required for fest merch?

Comment (Agrim1234) [reply app/src/main/java/com/macbitsgoa/events/aboutmac/AboutMacActivity.java]: this was redundant so i removed it

Comment (Rushi98) [reply app/src/main/java/com/macbitsgoa/events/aboutmac/AboutMacActivity.java]: remove all changes not required for solving this issue. 
Also I've rebased your repo master to origin master. You should pull it to local. (pull --hard if no local changes have done, else pull --rebase)

Comment (Rushi98) [reply app/src/main/java/com/macbitsgoa/events/aboutmac/AboutMacActivity.java]: also, rebase to latest origin master

Comment (Agrim1234) [reply app/src/main/java/com/macbitsgoa/events/aboutmac/AboutMacActivity.java]: git is showing that my branch is up to date with origin master

Comment (Agrim1234) [reply app/src/main/java/com/macbitsgoa/events/aboutmac/AboutMacActivity.java]: should i rebase my branch to upstream master

**Comment by Rushi98** [comment]
@Agrim1234 fix lint errors. You can generate the report locally by running `./gradlew clean lint`
